### ISSUE TITLE
Blog post: VTubing on Linux

### ISSUE THREAD

**Issue by kion-dgl** [DashGL/rabbit-hole#16]
How we do?

- https://flathub.org/apps/pro.vpup.vpuppr
- https://github.com/Hawkbat/VTubeStudioJS
- https://www.youtube.com/watch?v=F_QM4dm7FYk
- https://kitsune-one.itch.io/vtuber-webcam

### Option 1: Via Web Page

- https://3d.kalidoface.com/
- Note: Looks like it needs a VRM file, so I would need to commission one

### Option 2: Dock-it

- ROG Ally
- Win Mini
- Realistically it would be something I could dock with
- Win Mini makes more sense
- And potentially I could include other games and utilities

### Option 3: Headless

- Linux with HDMI Input
- Windows with HDMI Input
- Over the network with a Android/iPhone for tracking
- Not happy about an external device dependency 
- Basically like having a mini-pc and having to switch. 
- Z-13 flow or surface

### Option 4: Run in Linux

- Wine (god no)
- Emulated (web cam pass through)

**Comment by kion-dgl** [comment]
First impressions, a VRM model seems appealing. The cost of a vrm model can vary, but it needs to be taken into consideration compared to the price of a computer if it saves me from buying a deviated computer. 

For option 2 docks, I've been having a hard time with consistency getting any one dock to work with a range of devices. I would need to validate this works, and even then i wouldn't want to context switch to a different computer. 

For headless, or close dedicated. It's adds unwanted complexity to my desk that's already a mess. I'd prefer not to add something else that's going to be a distraction.

And last on linux, I prefer to avoid Wine. I'm only okay with wine when it's hidden from me entirely like on steam or lutris. For emulation, this is a cleaner version of the external approach, doesn't need separate hardware. But I still find it distracting.
